web/e2e/pages/landing/LandingPage.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import type { WebDriver, WebElement } from 'selenium-webdriver'
+import { By } from 'selenium-webdriver'
+
+export default class LandingPage {
+  driver: WebDriver
+  language: string
+
+  constructor (driver: WebDriver, langugage: 'de' | 'en' = 'de') {
+    this.driver = driver
+    this.language = langugage
+  }
+
+  ready (): Promise<void> {
+    return this.driver.get(`http://localhost:9000/landing/${this.language}`)
```

Comment:
I think we should make `http://localhost:9000` configurable in this task or the next one

Reply:
I kind of agree. But as I thought we agreed that with starting the server on a local port, even in the ci it is actually not necessary. What would be the actual advantage of setting `http://localhost:9000` configurable?

Reply:
If you run the tests in a separate selenium container then the app is no longer running on localhost form the perspective of selenium.

Then it is running e.g. on 172.17.0.5